whisker/src/pages/FlowLogsPage/index.tsx

```diff
@@ -68,20 +77,49 @@ const FlowLogsPage: React.FC = () => {
     } = useFlowLogsStream(urlFilterParams);
 
     const toast = useToast();
+    const selectedRowIdRef = React.useRef<string | null>(null);
+    const selectedRowRef = React.useRef<VirtualizedRow | null>(null);
+    const isWaitingRef = React.useRef<boolean>(false);
+    const hasStoppedRef = React.useRef<boolean>(false);
+    isWaitingRef.current = isWaiting;
+    hasStoppedRef.current = hasStoppedStreaming;
+
+    const onRowClicked = (row: VirtualizedRow) => {
+        selectedRowRef.current = row;
+
+        if (hasStoppedRef.current && !selectedRowIdRef.current) {
```

Comment:
Ah so here we check if user already stopped the stream and if its yes we don't do the rest of the onrowclick action.